src/services/rdsDbInstance/connections.ts

```diff
@@ -73,8 +91,173 @@ export default ({
     }
   }
 
+  /**
+   * Find Cloudwatch Logs
+   */
+  const cloudwatchLogs = data.find(
+    ({ name }) => name === services.cloudwatchLog
+  )
+  if (cloudwatchLogs?.data?.[region]) {
+    const cloudwatchLogsInRegion = cloudwatchLogs.data[region].filter(
+      ({ arn }: RawAwsLogGroup) =>
+        arn &&
+        EnhancedMonitoringResourceArn?.includes(
+          arn.substring(0, arn.length - 1)
+        )
+    )
+    if (!isEmpty(cloudwatchLogsInRegion)) {
+      for (const cloudwatchLog of cloudwatchLogsInRegion) {
+        connections.push({
+          id: cloudwatchLog.logGroupName,
+          resourceType: services.cloudwatchLog,
+          relation: 'child',
+          field: 'cloudwatchLogs',
+        })
+      }
+    }
+  }
+
+  /**
+   * Find Route53 Hosted Zone
+   */
+  const route53HostedZones = data.find(
+    ({ name }) => name === services.route53HostedZone
+  )
+  if (route53HostedZones?.data?.[globalRegionName]) {
+    const route53HostedZonesInRegion = route53HostedZones.data[
+      globalRegionName
+    ].filter(
+      ({ Id }: RawAwsRoute53HostedZone) =>
+        Endpoint?.HostedZoneId && Id.includes(Endpoint.HostedZoneId)
+    )
+    if (!isEmpty(route53HostedZonesInRegion)) {
+      for (const route53HostedZone of route53HostedZonesInRegion) {
+        connections.push({
+          id: getHostedZoneId(route53HostedZone.Id),
+          resourceType: services.route53HostedZone,
+          relation: 'child',
+          field: 'route53HostedZone',
+        })
+      }
+    }
+  }
+
+  /**
+   * Find KMS
+   * related to this RDS Cluster
+   */
+  const kms: {
+    name: string
+    data: { [property: string]: AwsKms[] }
+  } = data.find(({ name }) => name === services.kms)
+
+  if (kms?.data?.[region]) {
+    // set storage encryption kms key
+    let kmsInRegion: AwsKms[] = kms.data[region].filter(
+      ({ Arn }: AwsKms) => Arn === KmsKeyId
```

Comment:
should we also connect to the other kms keys here? `PerformanceInsightsKMSKeyId`, `ActivityStreamKmsKeyId`

Reply:
ahh nvm I see it below. It may make sense to just do one filter with multiple conditions.

Reply:
Yes, we haven't had many cases like this before. The downside of using only one attribute for KMS keys at the schema is that you will lose the information about the key purpose.

Reply:
yea its a trade off, in the past with these cases were just putting everything under the one connection field. @ckoning @kunovsky any preference here?

Reply:
I think it makes sense to keep the current structure and gather the different connections in one field. Then we can extend the connection object with an attribute that describes the type of relationship. What do you think @tyler-dunkel? it would be cleaner than have multiple fields